test/cpp_extensions/open_registration_extension/pytorch_openreg/csrc/OpenRegMem.cpp

```diff
@@ -266,10 +275,28 @@ void quantize_tensor_per_tensor_affine_privateuse1(
     // Just test the process, so do nothing
 }
 
+/* Notes:
+ *
+ * OpenReg is currently designed to simulate device memory through multiple
```

Comment:
nit: maybe worth mentioning that this is done on purpose to ensure we don't mistakenly poke at the "device's memory" from the main process.
And be able to simulate the same thing that happens with other accelerators: any metadata-only change is cpu-only (main process), any data change must go through to the device (other process) and any data transfer between the two is expensive (serializing the whole Tensor).

Reply:
Thank you, have updated it.